### PR TITLE
feat: bedrock support

### PR DESCRIPTION
**Pull request by AlexsJones**
Closes # 

## ✅ Checks

- [x ] My pull request adheres to the code style of this project
- [ x] My code requires changes to the documentation
- [x ] I have updated the documentation as required
- [ x] All the tests have passed

### CODE REVIEW

**Review by arbreezy — DISMISSED** [review]
lgtm, can't test it against bedrock though

I am a broken record but we also have to update the chart's crd templates (it's still manual)

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/k8sgpt-ai/k8sgpt-operator
-  newTag: feature-integration-support-4
+  newName: tibbar/k8sgpt-operator
```
Comment (JuHyung-Son) [inline config/manager/kustomization.yaml:7]: is this for testing in local env?

**Review by JuHyung-Son — DISMISSED** [review]
lgtm!

i left a question in kustomization image

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (AlexsJones) [reply config/manager/kustomization.yaml:7]: yeah, this isn't important. It gets overwritten

**Comment by AlexsJones** [comment]
> lgtm, can't test it against bedrock though
> 
> I am a broken record but we also have to update the chart's crd templates (it's still manual)

Thanks updated the chart 👍🏽

**Pull request merged**